R/shiny_prerendered.R

```diff
@@ -69,11 +69,12 @@ shiny_prerendered_app <- function(input_rmd, render_args) {
   }
 
   # attach dependencies to final html
-  html <- htmltools::attachDependencies(html, deps)
+  html_doc <- htmltools::htmlTemplate(text_ = html, document_ = TRUE)
+  html_doc <- htmltools::attachDependencies(html_doc, deps)
 
   # create shiny app
   shiny::shinyApp(
-    ui = function(req) html,
+    ui = function(req) html_doc,
```

Comment:
Unrelated to this change, but this can just be `ui = html_doc`, since the value is static

Reply:
@jcheng5 it seems it has been made a function to avoid caching of html_prerendered UI. See
https://github.com/rstudio/rmarkdown/commit/2de1b1de7c42f79bc40a798473b89aa46fef4e7b

Has it changed ? Should it be kept ?